generator/Parser/Tasks/Task.php

```diff
@@ -34,11 +34,19 @@ protected function getWrappedDefinitionProperty(string $selector): Collection
         return collect($property);
     }
 
+    /**
+     * Get the schema resource URL.
+     */
     protected function getResource(?array $schemaResource = null)
     {
         return $schemaResource['@id'] ?? $this->definition['@id'];
     }
 
+    /**
+     * Get the Schema resource name.
+     *
+     * Currently this omits the schema.org domain "http://scheme.org/"
```

Comment:
Doesn't schema.org at least in v9 switch officially to https?
Would a `trim(parse_url($url, PHP_URL_PATH)), '/')` work here more reliable? 
Or use the old `str_replace('http://schema.org/', '', $str)`.
From Developer-Experience/Maintenance perspective these two just feel better than `substr()`. 🤔
And possibly they are slower - but this code only runs on generation - so I don't care about some *ms* or even *s* in total.

And it's `schemA.org`^^
```suggestion
     * Currently this omits the schema.org domain "http://schema.org/"
```

Reply:
@mallardduck only this is an open task.
The others are only comments.

Reply:
> Doesn't schema.org at least in v9 switch officially to https?
Not strictly - they provide definition files with HTTP or HTTPS, but logically I think using HTTPS will make the most sense.

So I like the suggestion to just keep it simple on future reader of the code and using str_replace will be good. 😄

Reply:
Should be addressed now! 😄